cosmo_tester/framework/examples/nodecellar.py

```diff
@@ -68,18 +65,24 @@ def assert_mongodb_collector_data(self):
                     node_id=mongo_node_name
             )[0].id
 
-        try:
+        with self.manager.ssh() as fabric:
             # select metrics from the mongo collector explicitly to verify
             # it is working properly
-            query = 'select sum(value) from /{0}\.{1}\.{' \
-                    '2}\.mongo_connections_totalCreated/' \
-                .format(self.deployment_id, mongo_node_name,
-                        instance_id)
-            influxdb.query(query)
-        except Exception as e:
-            pytest.fail('monitoring events for {0} node instance '
-                        'with id {1} were not found on influxDB. error is: {2}'
-                        .format(mongo_node_name, instance_id, e))
+            result = fabric.run(
+                'curl -G "{url}" --data-urlencode '
+                '"q=select sum(value) from '
+                '/{dep}\.{mongo_node}\.{instance_id}\.'
+                'mongo_connections_totalCreated/"'.format(
+                    url=self.manager.influxdb_url,
+                    dep=self.deployment_id,
+                    mongo_node=mongo_node_name,
+                    instance_id=instance_id
+                ), quiet=True
+            )
+            if result == '[]':
```

Comment:
eh, maybe json.loads it after all? i suppose we want to know if the response format changes

Reply:
The result is not necessarily in JSON format, I don't think